### DISCUSSION TITLE
How to preview Markdown note

### DISCUSSION THREAD

**Discussion by ysslang** [TriliumNext/Trilium#2280] [Q&A] [answered]
Didn't find any instruction about how to render a markdown note. Is  this a trilium's feature or a ckeditor5's feature?

**Comment by abitofevrything** [comment]
If it is a code markdown note, you cannot preview it directly in Trilium. If it's a text note, formatting is generally applied dynamically as you type.

**Comment by zadam** [comment, marked as answer]
Hi, Trilium is primarily WYSIWYG editor and does not have 1st class support for Markdown features like preview etc.
